### DISCUSSION TITLE
`g?` does  not provide help?

### DISCUSSION THREAD

**Discussion by proofer** [mason-org/mason.nvim#589] [Q&A]
When I type `g?` with the Mason window up, nothing happens other  than `g?` being echoed in the statusline. Am I missing something?

**Comment by yutkat** [comment]
I think it would be better to open a Mason window and check with the following command.
`:nmap <buffer> g?`

If you set it up properly, it should look like this.
```lua
n  g?          *@<Lua 2041: ~/.local/share/nvim/site/pack/packer/opt/mason.nvim/lua/mason-core/ui/display.lua:340>
```

**Reply by proofer** [reply]
Yes, that's the output I get from the `nmap` command, except  that the first number after `*@<Lua ` is different from 2041 and is not the same each time.

**Comment by williamboman** [comment]
Maybe you have a `:h 'timeoutlen'` that is very low, or you're not pressing the sequence fast enough?

**Reply by proofer** [reply]
My `timeoutlen` is 500; I have no probems with other `g` commands or `<leader>` or other multi-key commands.

**Reply by williamboman** [reply]
Hm. Do you have a recording or screenshots? Is it echoed as a message or does it show up in the `'showcmd'` line? Is it reproducible with a minimal configuration?

**Reply by proofer** [reply]
Here's a verbal recording :) -- When I type `g`, "g" displays on the _right_ side of the command line, just like the first character of any multi-character command -- except, although the "g" remains until I type something else,  when I type my `<leader>` key, which is `<space>,` `<20>` displays for only a fraction of a second. In this respect `g` is like `z` or `f` or `t`. Then if the next character, after a delay of (presumably) any length, that I type is `?`, I now have `g?` displayed on the right side of the command line, which is otherwise empty. Nothing happens after that until I type another character. If that character is `<Enter>`, then `E21: Cannot make changes, 'modifiable' is off` displays on the left side of the command line. (If, without having typed `g0` I type `<Enter> `with the Mason window  up, nothing happens.)

Is there a reasonably easy way to create a minimal configuration that includes Mason? I'm using Packer.

**Reply by williamboman** [reply]
Since this only needs mason.nvim to be available in `'rtp'` you could clone the repo and starting a clean neovim instance with the git repo prepended to rtp:

```sh
$ cd mason.nvim
$ nvim --clean -c "set rtp^=$PWD" -c "lua require('mason').setup()"
```

Do the other keybinds work, like expanding a package via `<CR> (Enter)` or toggling views via `1 - 5`?

**Reply by proofer** [reply]
The issue is unchanged with that minimal config recommendation.

Yes, the other keybinds work. I have been using Mason for a while, but delayed reporting this issue because it's minor with an easy workaround -- referring to the end of the README.
